tasks/clean.yaml

```diff
@@ -0,0 +1,13 @@
+tasks:
```

Comment:
Only reason I didn't was then it could be optional. Wasn't sure if there are fringe cases where we might not want to delete those files or people didn't want to add the extra time it takes to delete the files to their tests. Can easily make the change if that's preferred.

Reply:
I think its still optional to call it regardless as long as its a separate task under `tasks/setup.yaml`. Someone could still include that file but not call the specific runner cleanup task I think?

Reply:
I see what you're getting at. I misinterpreted what you were saying and thought you meant add it to `.github/actions/setup/action.yaml`. I agree it makes more sense to add it to the `setup.yaml` and I will update the corresponding docs as well.